pristine-zen-garden.html

```diff
@@ -1,201 +1,154 @@
 <!DOCTYPE html>
 <html lang="en">
 <head>
-    <meta charset="utf-8">
-    <title>CSS Zen Garden: The Beauty of CSS Design</title>
+<meta charset="utf-8">
```

Comment:
Hey man, I think all of these meta tags etc could be indented to show they are nested inside <head>